topcoffea/modules/utils.py

```diff
@@ -123,6 +123,7 @@ def get_info(fname, tree_name = "Events"):
     raw_events = 0  # The raw number of entries as reported by TTree.num_entries
     gen_events = 0  # Number of gen events according to 'genEventCount' or set to raw_events if not found
     sow_events = 0  # Sum of weights
+    sow_lhe_wgts = 0 #Sume of LHE weights
```

Comment:
Could you explain a bit about what this is doing and/or why it's needed?

Reply:
This is needed for data files. Without this, the method would try to return sow_lhe_wgts without any declaration. Other option is to have conditional return statements based on is_Data

Reply:
Ah, ok. Can you fix the formatting of the comment (space between "#" and comment) and "Sume" -> "Sum" typo.